packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -252,7 +257,22 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 		<Stack>
 			<StackItem>
 				<Table size="extra-small" ref={tableRef}>
-					{DataRow("Container", containerState.id, columnSizing_unstable)}
+					{DataRow(
+						<>
```

Comment:
We may want to make this use `Stack` (horizontal) and `Stack.Item` and apply styling. We may also want to center the text vertically to line up better with the icon?

Reply:
Somehow I had missed this comment. The text seems to center vertically already:

![image](https://user-images.githubusercontent.com/716334/236318753-7d64307c-f2ff-4a98-8960-c1741e86aeb1.png)

Not sure exactly why we'd need to switch to Stack...?

Reply:
I think I was mostly suggesting that to keep structure in line with other components, but it looks like we have to migrate away from Stack anyways, so please disregard.

Reply:
One note though, now that you've shared the narrow-view screenshot: It's a bit hard to tell where one GUID ends and the next begins. I wonder if we want to increase spacing between entries and/or add separators? (Doesn't have to be in this PR)